### DISCUSSION TITLE
[Feature Request] Add a 'Go to Library' button on the 'Collections' page.

### DISCUSSION THREAD

**Discussion by flamesky80** [gotson/komga#1741] [Ideas]
If you open a comic from the 'Browse' tab, there's a button to go back. But if you open one from 'Collections', there's no button to get back to the library.

![image](https://github.com/user-attachments/assets/6af8a9d2-c2f5-4b7c-a467-e616c7dea13d)
![image](https://github.com/user-attachments/assets/3c628e88-d567-4b6f-8193-55fc9668b471)
![image](https://github.com/user-attachments/assets/92ab7959-b896-4e1a-9a2b-90ccebab50a9)
![image](https://github.com/user-attachments/assets/4c2a837f-5525-472b-8ae1-f515c5ba92c9)

**Comment by gotson** [comment]
It's not a button to go back, it's a button to go to the upper level. Just use the back button of your browser.

**Reply by flamesky80** [reply]
Please take a look at the picture and see what's different:
![operation](https://github.com/user-attachments/assets/94ebfba0-c885-4642-b484-1ef5740d64d8)

There‘’s not way to get back to the 'Collections' browser page.
Using the 'back' button in Chrome or Edge just returns me to the comic book page.

**Reply by flamesky80** [reply]
> Please take a look at the picture and see what's different: ![operation](https://private-user-images.githubusercontent.com/15180023/378674261-94ebfba0-c885-4642-b484-1ef5740d64d8.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAwMjQ4MDYsIm5iZiI6MTczMDAyNDUwNiwicGF0aCI6Ii8xNTE4MDAyMy8zNzg2NzQyNjEtOTRlYmZiYTAtYzg4NS00NjQyLWI0ODQtMWVmNTc0MGQ2NGQ4LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI3VDEwMjE0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzZWE4MjYxYTIxMjMyY2U4NDM0YmY4ZmNkNjU3YjEwZGQwMTNhM2ZkM2Y4Mzk4ODZjNTIxMTg1YWFiZmMxMTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.-riQm-5ailyyqC8R19egi1wvGH26nrLvzAN0Yn4eCQE)
> 
> There‘’s not way to get back to the 'Collections' browser page. Using the 'back' button in Chrome or Edge just returns me to the comic book page.

@gotson
